src/strategies/galxe-loyalty-points/examples.json

```
@@ -0,0 +1,23 @@
+[
+  {
+    "name": "Galxe users loyalty points",
+    "strategy": {
+      "name": "galxe-loyalty-points",
+      "params": {
+        "symbol": "PTS",
+        "spaceId": 263,
```

```suggestion
        "spaceId": 263
```
Tests are failing because of this